src/test/java/com/trello/boards/NewBoardTest.java

```diff
@@ -0,0 +1,124 @@
+package com.trello.boards;
+
+import com.trello.Waiters;
+
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.interactions.Actions;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.Test;
+
+import java.util.List;
+
+public class NewBoardTest {
+
+    String newBoardTitle = "Auto test - "+System.currentTimeMillis();
+
+    @AfterMethod
+    public void signOffFromTrello() {
+        System.setProperty("webdriver.chrome.driver", "C://Users//Office//Downloads//chromedriver_win32 (1)//chromedriver87.exe");
+        ChromeDriver driver = new ChromeDriver();
+
+        List<WebElement> openMemberMenu = driver.findElementsByXPath(".//button[contains(@aria-label, 'Open Member Menu')]");
+
+        if (openMemberMenu.size() > 0) {
+            openMemberMenu.get(0).click();
+            driver.findElementByLinkText("Sign Off").click();
+        }
+    }
+
+    @Test
+    public void searchBoardFieldCheck() {
```

Comment:
I don't think this test is necessary for automation in current state.
If you want to check search field, it will be better to create some board, and search it by correct and incorrect names.

Reply:
ok